### DISCUSSION TITLE
Can I drag file from tauri-app to another application?

### DISCUSSION THREAD

**Discussion by lomirus** [tauri-apps/tauri#8842] [Q&A] [answered]
Hi, I am making an file manager and now I want to drag a video file from my app to Potplayer (or any other players). Is it possible? I only find the solution about dragging files from other apps to tauri-app, not the other way around.

**Comment by lomirus** [comment, marked as answer]
Just found the solution. Seems I can simply write the path into `dataTransfer`:

```html
<div id="drag_source" draggable="true">Drag Me</div>
```

```javascript
const dragSource = document.querySelector('#drag_source')

dragSource.addEventListener("dragstart", function (event) {
    event.dataTransfer.setData("text", 'D:\\Videos\\abc.mp4');
});
```

**Reply by oopserian** [reply]
hi，how to solve the problem of dragging to the desktop?

**Reply by lomirus** [reply]
I only tested my answer for dragging a video file to Potplayer (or some other similar apps). As for dragging to the desktop, I have no solution yet.

**Reply by oopserian** [reply]
you can try this
https://github.com/crabnebula-dev/drag-rs

**Reply by creeper12356** [reply]
Hi, it seems that this plugin can only get the drop position, not the drop file path. 
Does somebody have a workaround for this issue?
